src/applications/hca/components/ApplicationDownloadLink.jsx

```diff
@@ -65,7 +65,7 @@ const ApplicationDownloadLink = ({ formConfig }) => {
         handlePdfDownload(blob);
         recordEvent({ event: 'hca-pdf-download--success' });
       } catch (error) {
-        setErrors(error.errors || []);
+        setErrors(error.errors);
```

Comment:
This shouldn't default to an empty array. Empty arrays are treated as if no error exists.